src/social_auth/backends/bitbucket.py

```diff
@@ -76,6 +76,7 @@ class BitbucketAuth(ConsumerBasedOAuth):
     AUTH_BACKEND = BitbucketBackend
     SETTINGS_KEY_NAME = 'BITBUCKET_CONSUMER_KEY'
     SETTINGS_SECRET_NAME = 'BITBUCKET_CONSUMER_SECRET'
+    DEFAULT_SCOPE = ['webhook', 'repository', 'issue']
```

Comment:
I hope that existing bitbucket authed repos with old scopes don't have problems with this